movie-mlops-project/preprocessing/preprocessing.py

```
@@ -0,0 +1,421 @@
+            df['is_summer_release'] = df['release_month'].isin([6,7,8]).astype(int)
+            df['is_holiday_release'] = df['release_month'].isin([11,12]).astype(int)
+            df['is_spring_release'] = df['release_month'].isin([3,4,5]).astype(int)
+        
+        # 장르 관련 특성
+        if 'genre_ids' in df.columns:
+            # genre_ids가 문자열인 경우 리스트로 변환
+            def parse_genre_ids(genre_ids):
+                if isinstance(genre_ids, str):
+                    try:
+                        return json.loads(genre_ids.replace("'", '"'))
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)